mod_reforged/hooks/skills/actives/taunt.nut

```diff
@@ -1,8 +1,48 @@
 ::Reforged.HooksMod.hook("scripts/skills/actives/taunt", function(q) {
+	q.m.DefenseModifierFraction <- 0.2;	// This percentage of the users resolve is removed from the targets defense
+	q.m.MaxRangeDefenseDebuff <- 1;
+
+	q.getTooltip = @(__original) function()
+	{
+		local ret = __original();
+
+		if (this.m.DefenseModifierFraction != 0)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "When used on an adjacent target, reduce its Melee Defense and Ranged Defense by " + ::MSU.Text.colorizeFraction(this.m.DefenseModifierFraction) + " of your current Resolve (" + ::MSU.Text.colorizeValue(this.calculateDefenseModifier()) + ")"
```

Comment:
This will also need special handling for nested tooltip of this effect now, because you put actor dependent value of the defense modifier in the tooltip. I'd suggest adding a `getNestedTooltip` function and check if the actor is the MSU dummy player and if yes then return a tooltip entry without that information. See for example my implementation of nested tooltip of the `rf_between_the_eyes` skill.